apps/cloud_composer/include/pcl/apps/cloud_composer/tools/impl/supervoxels.hpp

```diff
@@ -95,11 +95,12 @@ pcl::cloud_composer::SupervoxelsTool::performTemplatedAction (const QList <const
     std::map <std::uint32_t, typename pcl::Supervoxel<PointT>::Ptr > refined_supervoxel_clusters;
     super.refineSupervoxels (3, refined_supervoxel_clusters);
   
-    typename pcl::PointCloud<PointXYZRGBA>::Ptr color_segments;
-    color_segments= super.getColoredVoxelCloud ();
-    
-    CloudItem*  cloud_item_out = CloudItem::createCloudItemFromTemplate<PointXYZRGBA>(input_item->text(),color_segments);
- 
+    typename pcl::PointCloud<PointXYZL>::Ptr label_segments = super.getLabeledVoxelCloud ();
+    typename pcl::PointCloud<PointXYZRGBA>::Ptr color_segments(new pcl::PointCloud<PointXYZRGBA>);
```

Comment:
Do you need the `typename` keyword here? This should not be a dependent type anymore.